README.md

```diff
@@ -58,6 +59,20 @@ $ meteor add universe:i18n
 $ meteor npm install --save @types/meteor-universe-i18n
 ```
 
+## Migration to v2 / Breaking changes
+- We removed `locales` and `currency data` from our package. 
+  Delete from your code the following functions: `parseNumber`, `getLanguages`, `getCurrencyCodes`, `getCurrencySymbol`, `getLanguageName`, `getLanguageNativeName` and `isRTL`
+- Remove `_purify` option ([explanation](https://github.com/vazco/meteor-universe-i18n/issues/144#issuecomment-1006522371))
+- Remove `createTranslator` and `createReactiveTranslator`
+
+### React migration
+`meteor-universe-i18n` has no longer built-in React integration. However, we described [several approaches](https://github.com/vazco/meteor-universe-i18n/integration-with-react) to integrating React with our package. You need also to remove  `createComponent` and `getRefreshMixin` functions.
+
+
+### Blaze migration
+- Remove deprecated [universe-i18n-blaze](https://github.com/vazco/universe-i18n-blaze) package 
+- Add Blaze integration described in the [documentation](https://github.com/vazco/meteor-universe-i18n/#integration-with-blaze) below.
```

Comment:
I don't think it makes sense to have these as separate sections - listing them in one list with the points above should be enough.